graphqlmetrics/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.0
 
 require (
 	connectrpc.com/connect v1.16.2
-	github.com/ClickHouse/clickhouse-go/v2 v2.30.0
+	github.com/ClickHouse/clickhouse-go/v2 v2.32.2
```

Comment:
Since the ch-go had a breaking change and we needed to update it, I updated it through the parent library.